packages/theme-ui/src/merge.js

```diff
@@ -0,0 +1,24 @@
+import deepmerge from 'deepmerge'
+
+const canUseSymbol = typeof Symbol === 'function' && Symbol.for
```

Comment:
Is this legacy browser support?

Reply:
Yeah, it's derived from https://github.com/TehShrike/is-mergeable-object/blob/master/index.js which is the default for `deepmerge`. Since this is to support `React.forwardRef`, it might make sense to PR this upstream